python/lsst/sims/maf/plots/spatialPlotters.py

```diff
@@ -639,3 +640,93 @@ def __call__(self, metricValueIn, slicer, userPlotDict, fignum=None, ):
             cb1.solids.set_edgecolor("face")
         fig = plt.gcf()
         return fig.number
+
+
+class LambertSkyMap(BasePlotter):
+    """
+    Use basemap and contour to make a Lambertian projection.
+    Note that the plotDict can include a 'basemap' key with a dictionary of
+    kwargs to use with the call to Basemap.
+    """
+
+    def __init__(self):
+        self.plotType = 'SkyMap'
+        self.objectPlotter = False
+        self.defaultPlotDict = {'basemap':{'projection':'nplaea', 'boundinglat':20,
+                                           'lon_0':0., 'resolution':'l', 'celestial':True},
+                                'cbar':True, 'cmap':plt.cm.jet, 'levels':200,
+                                'cbarFormat':'%.2f','cbar_edge':True, 'zp':None,
+                                'normVal':None, 'percentileClip':False, 'colorMin':None,
+                                'colorMax':None, 'linewidths':0}
+
+    def __call__(self, metricValue, slicer, userPlotDict, fignum=None):
+
+        plotDict = {}
+        plotDict.update(self.defaultPlotDict)
+        plotDict.update(userPlotDict)
+
+        if plotDict['zp'] is not None :
+            metricValue = metricValue - plotDict['zp']
+        if plotDict['normVal'] is not None:
+            metricValue = metricValue/plotDict['normVal']
+
+        if plotDict['percentileClip']:
+            pcMin, pcMax = percentileClipping(metricValue.compressed(),
+                                              percentile=plotDict['percentileClip'])
+        if plotDict['colorMin'] is None and plotDict['percentileClip']:
+            plotDict['colorMin'] = pcMin
+        if plotDict['colorMax'] is None and plotDict['percentileClip']:
+            plotDict['colorMax'] = pcMax
+        if (plotDict['colorMin'] is not None) or (plotDict['colorMax'] is not None):
+            clims = [plotDict['colorMin'], plotDict['colorMax']]
+        else:
+            clims = None
+
+        # Make sure there is some range on the colorbar
+        if clims is None:
+            if metricValue.compressed().size > 0:
+                clims=[metricValue.compressed().min(), metricValue.compressed().max()]
+            else:
+                clims = [-1,1]
+            if clims[0] == clims[1]:
+                clims[0] =  clims[0]-1
+                clims[1] =  clims[1]+1
+
+        # Calculate the levels to use for the contour
+        if np.size(plotDict['levels']) > 1:
+            levels = plotDict['levels']
+        else:
+            step = (clims[1]-clims[0])/plotDict['levels']
+            levels = np.arange(clims[0],clims[1]+step, step)
+
+        fig = plt.figure(fignum)
+        ax = fig.add_subplot(111)
+
+        # Hide this extra dependency down here for now
+        # Note, this should be possible without basemap, but there are
+        # matplotlib bugs: http://stackoverflow.com/questions/31975303/matplotlib-tricontourf-with-an-axis-projection
+        from mpl_toolkits.basemap import Basemap
+
```

Comment:
I don't know if python cares (maybe it doesn't import a second time?) but you could move this import to the __init__ and then it should be called only once if we're reusing the __call__ method.  (maybe that doesn't happen enough to worry about it)